frontend/lib/map/map/map.dart

```diff
@@ -57,23 +55,27 @@ class _MapState extends State<Map> {
     return _mapboxMap;
   }
 
-  _onMapCreated(controller) async {
+  Future<void> _onMapCreated(MapboxMapController controller) async {
     this._controller = controller;
+    // zoom out to initial zoom level if can not get location
     await _controller
         .requestMyLocationLatLng()
-        .then((_) => _onUserLocationUpdated());
+        .then((_) {}, // like catchError, but do not enforce return type
```

Comment:
Why do we do nothing here?

Reply:
Because if we get there, that only means that we successfully retrieved to location and thus can stay zoomed in, so there is nothing to do. We only have to handle the error case, because then we have to access to the current location and thus should zoom out. As the comment says, there is an alternative to `then` for such cases, but it expects a location as return type, thus i did not use it.

Reply:
Ah, makes sense! Thanks for the explanation!

Reply:
Is there a difference to a try catch?
https://dart.dev/codelabs/async-await#example-async-and-await-with-try-catch
I think mixing the notation of `await/async` with `then,...` is confusing

Reply:
> I think mixing the notation of `await/async` with `then,...` is confusing

Agreed, I will change it